### PR TITLE
Change gke-large perf jobs to n1-standard-1 and run correctness suite on large-deploy

### PR DESCRIPTION
**Pull request by shyamjvs**
fyi - @wojtek-t 

/cc @krzyzacy - Could you lgtm?

### CODE REVIEW

**Comment by shyamjvs** [comment]
/hold cancel

Thanks.

**Comment by shyamjvs** [comment]
I changed it to 5k again, just to have a correctness run at that scale. Will switch back to 4k tomorrow morning CET to make some space for kubemark-scale presubmit.

File: jobs/config.json [hunk 1]
```diff
@@ -6652,7 +6652,7 @@
       "--gke-environment=test",
       "--gke-shape={\"default\":{\"Nodes\":4999,\"MachineType\":\"g1-small\"},\"heapster-pool\":{\"Nodes\":1,\"MachineType\":\"n1-standard-16\"}}",
       "--provider=gke",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --allowed-not-ready-nodes=50 --node-schedulable-timeout=60m --minStartupPods=8",
+      "--test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]|\\[DisabledForLargeClusters\\] --allowed-not-ready-nodes=50 --node-schedulable-timeout=60m --minStartupPods=8",
```
Comment (wojtek-t) [inline jobs/config.json]: Well - this is no longer "deploy" tests - for such purpose we have: "gke-scale-correctness" suite.
Can you please change that?

Comment (shyamjvs) [reply jobs/config.json]: I've sent https://github.com/kubernetes/test-infra/pull/6802

**Pull request merged**